build.py

```diff
@@ -1278,6 +1315,27 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu, target_mach
     pip3 install --upgrade numpy && \
     rm -rf /var/lib/apt/lists/*
 """
+    # Add dependencies needed for tensorrtllm backend
+    if "tensorrtllm" in backends:
+        be = "tensorrtllm"
+        import importlib.util
+
+        import requests
```

Comment:
Wondering why `import` statement is in function body? 

Outcome will be the same, if package installation missed function won't work. <br>

Having the `import` statement in the header of the body gives us an advantage to catch issue with configuration of the runtime environment without prerequisites or condition match.

Reply:
That's good point. I think the intention behind this was that we only want to import those packages when we need them. e.g. the `importlib.util` and `requests` are only needed when building FT and TRT-LLM backends.

Reply:
I don't see any specific PIP package installation for given `import` statement assume it safe to move to the top.

Reply:
Addressed.